<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QLabel" name="lbl_selected_bus">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>620</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>BUS STOP: </string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>681</width>
     <height>421</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>679</width>
      <height>419</height>
     </rect>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>661</width>
       <height>401</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="scrollArea_layout"/>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_Next">
   <property name="geometry">
    <rect>
     <x>694</x>
     <y>52</y>
     <width>100</width>
     <height>420</height>
    </rect>
   </property>
   <property name="text">
    <string>NEXT</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
